src/ralph/operations/tests/test_admin.py

```diff
@@ -0,0 +1,16 @@
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+
+from ralph.operations.models import OperationType
+from ralph.operations.tests.factories import OperationFactory
+
+from ralph.tests.mixins import ClientMixin
+
+
+class OperationAdminViewTest(ClientMixin, TestCase):
+    def test_operation_changelist_should_run_2_queries(self):
+        OperationFactory.create_batch(5)
```

Comment:
unfortunately, `OperationFactory` does not assign service_env and config. path, so this doesn't really test your code :)

Reply:
done